web/app/static/js/controllers/clusterController.js

```diff
@@ -554,13 +554,13 @@ angular.module('uluwatuControllers').controller('clusterController', ['$scope',
                     $rootScope.activeClusterBlueprint = $filter('filter')($rootScope.blueprints, {
                         id: $rootScope.activeCluster.blueprintId.toString()
                     }, true)[0];
-                    if ($rootScope.activeCluster.credentialId) {
+                    if ((typeof($rootScope.activeCluster.cloudPlatform) === "undefined" || !$rootScope.activeCluster.cloudPlatform) && $rootScope.activeCluster.credentialId) {
                         $rootScope.activeClusterCredential = $filter('filter')($rootScope.credentials, {
                             id: $rootScope.activeCluster.credentialId
                         }, true)[0];
-                        $rootScope.activeCluster.cloudPlatform = $rootScope.activeClusterCredential.cloudPlatform
-                    } else {
-                        $rootScope.activeCluster.cloudPlatform = $rootScope.activeCluster.orchestrator.type
```

Comment:
That make no sense, because originally cloudPlatform was "OPENSTACK" on mine, and this line replaces it to "SALT" which isn't a cloudPlatform.